tstest/natlab/vnet/vnet.go

```diff
@@ -2185,13 +2153,34 @@ func mkPacket(layers ...gopacket.SerializableLayer) []byte {
 }
 
 func mkPacketErr(ll ...gopacket.SerializableLayer) ([]byte, error) {
```

Comment:
Add a comment that says the ordering matters. It's obvious but still.

Reply:
Also document defaulting behavior?

Reply:
done